src/modules/salesforce/salesforce.module.ts

```diff
@@ -0,0 +1,41 @@
+import https from 'https';
+import crypto from 'crypto';
+
+import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { SalesforceConfig, KEY as SALESFORCE_CONFIG_KEY } from '@ukef/config/salesforce.config';
+import { HttpModule } from '@ukef/modules/http/http.module';
+
+import { SalesforceService } from './salesforce.service';
+
+@Module({
+  imports: [
+    HttpModule.registerAsync({
+      imports: [ConfigModule],
+      inject: [ConfigService],
+      useFactory: (configService: ConfigService) => {
+        const { baseUrl, clientId, clientSecret, username, password, accessUrl, maxRedirects, timeout } = configService.get<SalesforceConfig>(SALESFORCE_CONFIG_KEY);
+        return {
+          baseURL: baseUrl,
+          clientId,
+          clientSecret,
+          username,
+          password,
+          accessURL: accessUrl,
+          maxRedirects,
+          timeout,
+          // TODO: APIM-471 - cleanup rejectUnauthorized when Informatica SSL issue is resolved
+          // "rejectUnauthorized: false" is just for local DEV laptop environment, not for DEV/PROD.
+          // to ignore https issues, enable agent and rejectUnauthorized:false bellow.
+          httpsAgent: new https.Agent({
+            // Allow self signed negotiations
+            // rejectUnauthorized: false,
```

Comment:
this needs to be uncommented to work locally, but on prod it should work fine... I'm not that happy with this state of things so if there's a better way that allows it to work everywhere without tweaking the code that'd be ideal.